### PR TITLE
GET /api/management/v2/devauth/devices endpoint

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Issues: https://tracker.mender.io/browse/MEN-1982

### CODE REVIEW

File: api/http/api_devauth.go [hunk 1]
```diff
@@ -242,7 +248,49 @@ func (d *DevAuthApiHandlers) GetDevicesHandler(w rest.ResponseWriter, r *rest.Re
 
 	skip := (page - 1) * perPage
 	limit := perPage + 1
-	devs, err := d.devAuth.GetDevices(ctx, uint(skip), uint(limit))
+	devs, err := d.devAuth.GetDevices(ctx, uint(skip), uint(limit), store.DeviceFilter{})
```
Comment (mchalski) [inline api/http/api_devauth.go]: this returns `model.Device` structs, not sure if it fits 100%.

if i'm seeing correctly, the doc says the field is name `identity_data`, but model has `id_data`:
https://github.com/mendersoftware/deviceauth/blob/master/model/device.go#L34
https://github.com/mendersoftware/deviceauth/blob/master/docs/management_api_v2.yml#L397

Comment (kjaskiewiczz) [reply api/http/api_devauth.go]: fixed

**Comment by kjaskiewiczz** [comment]
integration: https://mender-jenkins.mender.io/job/mender-builder/1722/

**Comment by kjaskiewiczz** [comment]
integration: https://mender-jenkins.mender.io/job/mender-builder/1722/

**Comment by kjaskiewiczz** [comment]
integration: https://mender-jenkins.mender.io/job/mender-builder/1722/

**Comment by kjaskiewiczz** [comment]
integration: https://mender-jenkins.mender.io/job/mender-builder/1722/

**Pull request merged**